boards/DIM/Test/Src/Test_StateMachine.cpp

```diff
@@ -188,6 +189,7 @@ TEST_F(DimStateMachineTest, check_drive_state_is_broadcasted_over_can)
         App_CanTx_GetPeriodicSignal_STATE(can_tx_interface));
 }
 
+// DIM_9
```

Comment:
We should consistent about using `-` vs `_` in these specs eh? I think `-` better matches the specs markdown doc.

Reply:
Definitely unintentional, will fix

Reply:
Should be fixed now!